### ISSUE TITLE
Time statistics / visualisations.

### ISSUE THREAD

**Issue by siamezzze** [kwadraterry/GPGPU-LUT#15]
Prepare statistics, graphics, etc. for report.

**Comment by siamezzze** [comment]
Please add similar info for MATLAB code. 

I tried to do this myself, but it crashes on test images. I suspect that is because the code is not ready yet (judging from the fact you have not closed #5 ), so I'm not opening a separate bug, but feel free to do that yourself.




[You can download test images from here](https://drive.google.com/file/d/0B7aQ64Q4Sx3POTVnZWsySUlsTEk/view?usp=sharing) or use this code to generate them for yourself:

```
declare -a sizes=(1 5 10 15 25 50 75 100 150 250 500 1000 2500 5000 7500)

for size in "${sizes[@]}"; do echo "$size"; name=$(printf 'data/plasma%04u.png' "$size"); convert -size "$size"x"$size" plasma: $name; done;
```

**Comment by siamezzze** [comment]
Added MATLAB info.
